test/unit/cute/intel_xe/mma.cpp

```diff
@@ -30,30 +30,23 @@
  *
  **************************************************************************************************/
 
-#include "cutlass/util/GPU_Clock.hpp"
-#include "cutlass/util/print_error.hpp"
-
 #include <cute/tensor.hpp>
 #include <sycl/sycl.hpp>
 #include <syclcompat.hpp>
+#include <syclcompat/launch_policy.hpp>
```

Comment:
`syclcompat.hpp` should also include the `launch_policy.hpp` file,
so this include is not required